content/2021-04-16-newsletter-28.md

```diff
@@ -84,6 +86,7 @@ TODO: Put all links for content here.
 [device-driver]: https://crates.io/crates/device-driver
 [embedded-foo 1.0.0 released]: https://example.com/blog/...
 [embedded-graphics 0.7.0-beta.1 released]: https://crates.io/crates/embedded-graphics/0.7.0-beta.1
+[svd2rust 0.18]: https://crates.io/crates/svd2rust
```

Comment:
```suggestion
[svd2rust 0.18.0]: https://crates.io/crates/svd2rust
```
Reference and link name are not lining up.

Reply:
Aww, nuts. I changed one at the last minute and didn't fix the link. Will update.